spec/03-plugins/06-statsd/01-log_spec.lua

```diff
@@ -906,7 +924,7 @@ for _, strategy in helpers.each_strategy() do
       end)
 
       it("logs over UDP with default metrics with dogstatsd tag_style", function()
-        local metrics_count = DEFAULT_METRICS_COUNT - 6
+        local metrics_count = DEFAULT_METRICS_COUNT + shdict_count * 2 - 6
```

Comment:
`DEFAULT_METRICS_COUNT - 6` was already difficult to understand, and adding `shdict_count * 2` everywhere made it even worse.  Please define a couple of clearly named and commented variables as an upvalue at the top of the test and use that instead of calculating a `metrics_count` over and over again.

Reply:
1. I think this is very clear, clearly telling the reader of the test code how the expected number of metrics is calculated.
2. `shdict_count` will dynamically change in test running (because `nginx.conf` may be changed in some cases), So recalculating this count is make sense.

Reply:
As your wish, I have added it. Please review it again.